# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../samba/fix_smbadduser_csh_usage.patch
# Copyright (C) 2006 - 2009 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./source3/smbadduser.in.orig	2006-01-16 22:50:47.000000000 +0000
+++ ./source3/smbadduser.in	2006-01-16 22:51:12.000000000 +0000
@@ -3,11 +3,11 @@
 # smbadduser - Written by Mike Zakharoff
 #
 
-prefix=@prefix@
-exec_prefix=@exec_prefix@
-LIBDIR=@libdir@
-PRIVATEDIR=@privatedir@
-CONFIGDIR=@configdir@
+set prefix=@prefix@
+set exec_prefix=@exec_prefix@
+set LIBDIR=@libdir@
+set PRIVATEDIR=@privatedir@
+set CONFIGDIR=@configdir@
 
 unalias *
 set path = ($path /usr/local/samba/bin)
